types/koa-generic-session/index.d.ts

```diff
@@ -50,5 +51,13 @@ declare namespace koaSession {
     const MemoryStore: SessionStore;
 }
 
+declare module 'koa' {
+    interface Context {
+        session: koaSession.Session|null;
+        sessionSave: boolean|null;
+        regenerateSession(): Generator;
```

Comment:
Where is `Generator` defined? It either needs to be defined here or imported from somewhere.